File: diet/src/main/java/com/example/diet/Domain/RegisterInfo.java
package com.example.diet.Domain;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RegisterInfo {
    private String usr_id;
    private String usr_phone;
    private String usr_password;
    private String usr_name;
    private boolean usr_sex;
    private String usr_height;
    private String usr_weight;
    private Long usr_age;
    private String tar_weight;
    private String tar_time;
}

File: frontend/DayDiet_google/app/src/main/java/com/example/daydiet/model/Domain/RegisterInfo.kt
package com.example.daydiet.model.entity

data class RegisterInfo(
    val usr_id : Long,
    val usr_phone : String,
    val usr_password : String,
    val usr_name : String,
    val usr_sex : Boolean,
    val usr_height : String,
    val usr_weight : String,
    val usr_age : Long,
    val tar_weight : String,
    val tar_time : String
)

// 获取一个RegisterInfo
data class InfoResponse(var data:RegisterInfo ): BaseResponse() {}
